src/test/rbd_mirror/image_sync/test_mock_ImageCopyRequest.cc

```diff
@@ -31,11 +42,11 @@ namespace mirror {
 namespace image_sync {
 
 template <>
-struct ObjectCopyRequest<librbd::MockImageCtx> {
+struct ObjectCopyRequest<librbd::MockTestImageCtx> {
```

Comment:
oh, i got what you meant. thought we needed to have `ObjectCopyRequest<librbd:: MockImageCtx >`